chef/data_bags/crowbar/bc-template-crowbar.json

```diff
@@ -29,19 +29,25 @@
   },
   "deployment": {
     "crowbar": {
-      "crowbar-revision": 0,
+      "crowbar-revision": 1,
       "crowbar-applied": false,
       "element_states": {
-        "crowbar": [ "all" ]
+        "crowbar": [ "all" ],
+        "crowbar-upgrade": [ "ready", "crowbar-upgrade" ]
```

Comment:
I think in that case you need "applying" (I would put "readying" too just because we usually put the 3 values)